web/components/Nav/Nav.tsx

```diff
@@ -21,6 +22,28 @@ export const Nav = ({ onFrontPage }: NavProps) => {
   const toggleMenu = () => setMenuOpened(!menuOpened);
   const closeMenu = () => setMenuOpened(false);
 
+  const BasicMenuItem = ({
+    urlPath,
+    label,
+  }: {
+    urlPath: string;
+    label: string;
+  }) => {
+    const isCurrentPage = urlPath === currentPath;
```

Comment:
Nit(personal preference): this constant could be inlined, which allows dropping the surrounding curlies and the explicit `return`.